glossary/index.md

```diff
@@ -2,6 +2,8 @@
 # SPDX-License-Identifier: CC0-1.0
 # SPDX-FileCopyrightText: 2018-2023 The Foundation for Public Code <info@publiccode.net>
 type: Index
+redirect_from:
+    - /glossary/public-member-definition
```

Comment:
I think this is a good redirect. Should we also have a sentence in the top saying something like "if you cannot find the term you were looking for, we may have deleted it and you were redirected to the top of the glossary"

Reply:
Moved to #1350